dawn/.eslintrc

```diff
@@ -33,6 +33,8 @@
     "react/no-unused-prop-types": 0,
     "no-useless-escape": 0,
     "import/prefer-default-export": 0,
-    "import/no-extraneous-dependencies": 0
+    "import/no-extraneous-dependencies": 0,
+    "no-empty": 0,
```

Comment:
Wouldn't it make more sense to just disable the rule on a case-by-case basis instead of for the entire project? These rules are kinda useful to keep enabled.